themes/_core/js/product.js

```diff
@@ -54,10 +54,10 @@ function getProductUpdateUrl() {
 
   if (
     prestashop !== null
-    && prestashop.urls !== null
-    && prestashop.urls.pages !== null
-    && prestashop.urls.pages.product !== ''
-    && prestashop.urls.pages.product !== null
+        && prestashop.urls !== null
+        && prestashop.urls.pages !== null
+        && prestashop.urls.pages.product !== ''
+        && prestashop.urls.pages.product !== null
```

Comment:
Hello,

These tabs are really unnecessary, right? Same for every other next

Reply:
Hello,
I just checked, indeed it's a strange, automatic formatting that has trouble happening. I just corrected it.